index.css

```diff
@@ -91,6 +100,37 @@ body {
   padding-right: 8%;
 }
 
+.connect {
+  font-weight: 500px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7b86ece6;
+  font-style: normal;
+  padding-left: 10%;
+  margin-top: 2%;
+}
+
+.main-heading {
+  padding-left: 75%;
+  padding-right: 12%;
+  margin-top: 0;
+  color: #115ae1cb;
+  font-weight: 7%;
+  font-size: 80%;
+  font-style: normal;
+  line-height: 52px;
+}
+
+.main-paragraph {
+  padding-left: 2%;
+  margin-bottom: 0.5%;
+  color: #344563;
+  font-weight: 7%;
+  font-style: normal;
+  font-size: 75%;
+  line-height: 24px;
+}
+
 @media screen and (min-width: 408px) {
```

Comment:
- [ ] Good job implementing responsive breakpoints. I suggest using `375px` for the mobile version minimum and `768px` for the desktop version minimum per the project requirements. 😃